pages/style-guide/html.md

```diff
@@ -4,64 +4,110 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
 
-```
-<!doctype html>
-```
+The markup should be valid HTML5 and it must pass the automated testing we use via grunt-html.
```

Comment:
`Use CSSLint to detect errors and potential problems. Most jQuery projects have a Grunt task for linting all CSS files: grunt csslint. The options for CSSLint are stored in a .csslintrc file.`
Like this? If yes, I should ask if there is a Grunt task for linting all html files?